### PR TITLE
Onboarding cluster-proxy-addon to MCE

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

https://github.com/stolostron/backlog/issues/23539

### CODE REVIEW

**Comment by xuezhaojun** [comment]
Get `Dockerfile build strategy has failed.` error. Is this requires CICD to change anything in the downstream building?

File: ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.1.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    dependencies:
+      SOURCE_IMAGE_REF: cluster-proxy-addon
+    env:
+      IMAGE_REPO: cluster-proxy-addon
+      REGISTRY_ORG: stolostron
+    test:
+    - as: publish
+      commands: |-
+        export OSCI_IMAGE_REMOTE_REPO_SRC="registry.ci.openshift.org/stolostron"
+        export OSCI_PUBLISH_DELAY="0"
```
Comment (zhujian7) [inline ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.1.yaml:91]: export OSCI_PIPELINE_PRODUCT_PREFIX=backplane
        export OSCI_PIPELINE_REPO=backplane-pipeline

File: ci-operator/config/stolostron/cluster-proxy/stolostron-cluster-proxy-backplane-2.1.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      SOURCE_IMAGE_REF: cluster-proxy
+    env:
+      IMAGE_REPO: cluster-proxy
+      REGISTRY_ORG: stolostron
+    test:
+    - as: publish
+      commands: |-
+        export OSCI_IMAGE_REMOTE_REPO_SRC="registry.ci.openshift.org/stolostron"
+        export SELF="make -f Makefile.prow"
+        export OSCI_PUBLISH_DELAY="0"
```
Comment (zhujian7) [inline ci-operator/config/stolostron/cluster-proxy/stolostron-cluster-proxy-backplane-2.1.yaml:70]: export OSCI_PIPELINE_PRODUCT_PREFIX=backplane
        export OSCI_PIPELINE_REPO=backplane-pipeline

File: ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.1.yaml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      commands: |
+        KUBECTL=oc
+        EXAMPLE_IMAGE_NAME=${IMAGE_FORMAT//\/stable:\$\{component\}//pipeline:cluster-proxy-addon}
+        export KUBECTL EXAMPLE_IMAGE_NAME
+        make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-aws
```
Comment (zhujian7) [inline ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.1.yaml:78]: Using `generic-claim` workflow can make the e2e faster. https://github.com/openshift/release/pull/28332

File: ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.1.yaml (hunk 1 above)
Comment (xuezhaojun) [reply ci-operator/config/stolostron/cluster-proxy-addon/stolostron-cluster-proxy-addon-backplane-2.1.yaml:98]: Added.

File: ci-operator/config/stolostron/cluster-proxy/stolostron-cluster-proxy-backplane-2.1.yaml (hunk 2 above)
Comment (xuezhaojun) [reply ci-operator/config/stolostron/cluster-proxy/stolostron-cluster-proxy-backplane-2.1.yaml:70]: Added.

**Comment by xuezhaojun** [comment]
@zhujian7 Comments are resolved. PTAL.

**Pull request merged**